app/decorators/models/solidus_payment_method_by_zone/spree/order_decorator.rb

```diff
@@ -0,0 +1,12 @@
+module SolidusPaymentMethodByZone
+  module Spree
+    module OrderDecorator
+      def available_payment_methods
+        @available_payment_methods ||= super.left_joins(:payment_method_zone)
+                                           .where(spree_payment_method_zones: {zone_id: [nil] + ::Spree::Zone.for_address(ship_address).pluck(&:id)})
```

Comment:
could you add spaces inside this hash:
`{ zone_id: [nil] + ::Spree::Zone.for_address(ship_address).pluck(&:id) })`